src/corefx/io/corefx_io.cpp

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#include "corefx_io.h"
+#include <sys/stat.h>
+
+#if HAVE_STAT64 && !(defined(__APPLE__) && defined(_AMD64_))
+#    define stat_ stat64
+#    define fstat_ fstat64
+#else
+#   define stat_ stat
+#   define fstat_ fstat
+#endif
+
+static void ConvertFileStats(const struct stat_& src, FileStats* dst)
+{
+    dst->Flags = FILESTATS_FLAGS_NONE;
+    dst->Mode = src.st_mode;
```

Comment:
I think we should declare these constants here even if they are fixed in practice. It's good discipline to follow the pattern we definitely need elsewhere.